### PR TITLE
SGZ

### PR DESCRIPTION
**Pull request by da-wad**
Do not allow for confusion of compression algorithm used. Instead use .sgz which sounds like a natural progression from .sgy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,31 +36,36 @@ compared to n_traces disk blocks for SEG-Y
...
-    converter.run("out-advanced.sz", bits_per_voxel=2, 
+    # Create a "z-slice optimized" SGZ file
+    converter.run("out-advanced.sgz", bits_per_voxel=2, 
                   blockshape=(64, 64, 4))
+# Convert back to SEG-Y
+with SgzConverter("out-standard.sgz") as converter:
+    converter.convert_to_segy("recovered.sgy")
 ```
 
 #### Read an SZ file ####
```
Comment (lindstro) [inline README.md]: Should be Read an SGZ file?

File: README.md [hunk 2]
```diff
@@ -36,31 +36,36 @@ compared to n_traces disk blocks for SEG-Y
...
 ```python
 from seismic_zfp.read import SzReader
```
Comment (lindstro) [inline README.md]: SgzReader?

File: README.md [hunk 3]
```diff
@@ -72,21 +77,14 @@ with SzReader("in.sz") as reader:
 #### Use segyio-like interface to read SZ files ####
```
Comment (lindstro) [inline README.md]: SGZ files?

File: README.md (hunk 1 above)
Comment (da-wad) [reply README.md]: True.

File: README.md (hunk 2 above)
Comment (da-wad) [reply README.md]: True.

File: README.md (hunk 3 above)
Comment (da-wad) [reply README.md]: True.

**Comment by da-wad** [comment]
My PyCharm crashes on editing markdown... thanks for spotting the ones it apparently missed.

**Pull request merged**